docs/tools/code-extension.mdx

```diff
@@ -31,8 +47,142 @@ To compile the `.lf` source, open the command palette (<kbd>Ctrl</kbd> + <kbd>Sh
 You can also build and immediately afterwards run your code by opening the command palette (<kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>P</kbd>) and then entering `Lingua Franca: Build and Run`.
 Running the code can also be done from the VS Code terminal by executing the generated file in `./bin`.
 
+### Packages Explorer
+
+The **Lingua Franca Packages Explorer** can be accessed by clicking on the **LF icon** in the activity bar on the left side of the screen. Once opened, the **Packages Explorer** displays a **Tree View** with the following structure:
+
+```
+├── LF Project
+│   ├── Installed Packages
+│   ├── Local Libraries
+└── └── Source Files
+```
+
+- [**Installed Packages**](#installed-packages): Lists libraries installed via the Lingo Package Manager, located in the `build/lfc_include` directory (if any).
+  
+- [**Local Libraries**](#local-libraries): Displays locally defined libraries (e.g., reusable reactors), located in the `src/lib` directory.
+  
+- [**Source Files**](#source-files): Contains the LF source files created by the developer, located in the `src` directory.
+
+The **Source Files** section is always present as it reflects the main LF files in the project. However, the **Installed Packages** and **Local Libraries** sections appear only if the respective directories and files exist in the workspace.
+
+Hovering over the **LF Project** item reveals a terminal icon. Clicking this icon opens a terminal window at the project's root directory, allowing you to execute commands directly within that directory.
+
+#### Installed Packages
+
+The **Installed Packages** section lists libraries installed using the [Lingo Package Manager](https://github.com/lf-lang/lingo). Developers can use the Lingo Package Manager to retrieve and install LF programs from repositories such as the [Lingua Franca Community Organization](https://github.com/LF-Community).
+
+To install libraries:
+1. Configure the `Lingo.toml` file with the desired libraries.
+2. Run `lingo build` to download the specified dependencies.
+
+Once the libraries are installed, they will appear in the `{project_name}/build/lfc_include/` directory. The **Installed Packages** section will then be structured as follows:
+
+```
+├── LF Project
+│   ├── Installed Packages
+│   │  ├── installed_library/
+│   │  │   ├── File_1.lf
+│   │  │   │   ├── Rusable_Reactor_1.lf
+│   │  │   │   ├── Rusable_Reactor_2.lf
+│   │  │   ├── File_2.lf
+│   │  │   │   ├── Rusable_Reactor_1.lf
+│   │  │   │   ├── Rusable_Reactor_2.lf
+...
+```
+
+In this structure:
+- **LF Project**: Represents the root folder of the main project.
```

Comment:
It seems like we mix the term "package" and "project" a lot. Maybe we should only use one?

Reply:
The `project` refers to the Lingua Franca project, while `packages` are the libraries installed and defined within it

Reply:
Well, to me, packages and projects are kind of the same. Once you create one, you can publish it, and someone else can use it. It looks like Rust also uses "package" and "project" interchangeably: 

See https://doc.rust-lang.org/cargo/guide/creating-a-new-project.html; in this definition they shed some light on the relationship between project and package: https://doc.rust-lang.org/cargo/appendix/glossary.html#package

Note that the "package explorer" to me is a view into the current package (not an explorer of package found inside of the current project).

Reply:
In a way, it's a bit ambiguous since they say: 
> ### [Project](https://doc.rust-lang.org/cargo/appendix/glossary.html#project)  
> Another name for a [package](https://doc.rust-lang.org/cargo/appendix/glossary.html#package).

However, it makes sense to me to standardize the use of `package` instead of `project`. But in doing so (as in Rust), the word `package` would essentially become another name for `project`, right?

Reply:
Jep. I think the most important thing here is to all agree on what we mean when we use these terms. I like the Rust approach of providing definitions; we should do the same.

Reply:
I agree. In that case, adding a `Glossary` page would be a great solution

Reply:
For now, should I add the `Glossary` with the definition of `LF Package` under the `Notes` section?

Reply:
A new page would be better. We have lots of definitions that can use explaining.